docs/production/installing-with-npm.md

```diff
@@ -14,138 +14,110 @@ eleventyNavigation:
 
-Follow either [Recommended solution](#recommended-solution) or [Alternative solution](#alternative-solution).
+1. Copy the:
 
-### Recommended solution
+- `/node_modules/@ministryofjustice/frontend/moj/assets/images` contents to `<YOUR-APP>/assets/images`
+- `/node_modules/govuk-frontend/dist/govuk/assets/images` contents to `<YOUR-APP>/assets/images`
+- `/node_modules/govuk-frontend/dist/govuk/assets/fonts` contents to `<YOUR-APP>/assets/fonts`
 
-Make `/node_modules/@ministryofjustice/frontend/moj/assets` available to your project by routing requests for your assets folder there.
+2. Run your application, then use [the Fonts tab in Firefox Page Inspector](https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector/How_to/Edit_fonts#The_Fonts_tab) to check the accordion is using the GDS Transport font.
 
-For example, if your project uses [express.js](https://expressjs.com/), below is a code sample you could add to your configuration:
+In your live application, we recommend [using an automated task or your build pipeline](/production/import-font-and-image-assets/) instead of copying the files manually.
 
-```JS
-app.use('/assets', express.static(path.join(__dirname, '/node_modules/@ministryofjustice/frontend/moj/assets')))
-```
+### Get the JavaScript working
 
-### Alternative solution
+1. Add the following to the top of the `<body class="govuk-template__body">` section of your page template:
 
-Manually copy the images and fonts from `/node_modules/@ministryofjustice/frontend/moj/assets` into a public facing directory in your project. Ideally copying the files to your project should be an automated task or part of your build pipeline to ensure that the MoJ Frontend assets stay up-to-date.
+   ```html
+   <script>document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');</script>
+   ```
 
-The default paths used for assets are `assets/images` and `assets/fonts`. **If your asset folders follow this structure, you will not need to complete the following steps.**
+2. Copy both `/node_modules/@ministryofjustice/frontend/moj/moj-frontend.min.js` and `/node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.js` files into your application.
 
-To use different asset paths, set `$govuk-assets-path`, `$govuk-images-path` and `$govuk-fonts-path` in your project Sass file to point to the relevant directories in your project (this will override the defaults set in `/node_modules/@ministryofjustice/frontend/moj/settings/_assets.scss`). Make sure you do this in Sass before importing `@ministryofjustice/frontend` into your project - see [Importing styles](#importing-styles).
+   You will also need to install and serve [jQuery](https://jquery.com/).
 
-Example 1:
+3. Import the files before the closing `</body>` tag of your page template, then run the `initAll` functions to initialise all the components. For example:
```

Comment:
Is there a reason we say to put them at the bottom of the body element? I thought that was an anti-pattern nowadays as it prevents the preload scanner/parser from discovering the scripts early and `script type="module"` is `defer` by default.